lib/transbank/sdk/patpass/patpass_by_webpay/patpass_by_webpay.rb

```diff
@@ -1,10 +1,12 @@
 require 'transbank/sdk/patpass/patpass_by_webpay/base'
 require 'transbank/sdk/patpass/patpass_by_webpay/options'
 require 'transbank/sdk/patpass/errors/patpass_error'
+require 'transbank/sdk/patpass/errors/integration_type_error'
```

Comment:
You already `require`d this in patpass_by_webpay_errors

Reply:
Fixed